docs/api/routing.rst

```diff
@@ -394,6 +394,60 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other architectures. Consider
+the task of building an API for a calculator which can both add and subtract two numbers. You could implement the 
+following:
+
+.. code:: python
+
+    class Add():
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) + int(req.get_param('y')))
+            resp.status = falcon.HTTP_200
+
+    class Subtract(object):
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) - int(req.get_param('y')))
+            resp.status = falcon.HTTP_200
+
+    add = Add()
+    subtract = Subtract()
+    api = falcon.API()
+    api.add_route('/add', add)
+    api.add_route('/subtract', subtract)
+
+However, in addition to being slightly verbose it's also logically a bit strange because adding and
```

Comment:
This paragraph sounds a bit colloquial; ideally, it could be edited to conform to the @kgriffs style™ :smiling_imp: 
But style-wise it passes my review _as-is_.

Reply:
You know, as I was writing this, I was actually wondering where the line was between technical and colloquial for Falcon docs, ha. I can rework to be less casual.

Reply:
Reworded a bit, let me know what you think.